processing/pipeline.py

```diff
@@ -835,9 +838,9 @@ def run_sherpa_optimize(fun, **kwargs):
                                 name=name, argument_type=ArgumentType.VALUE, data=value
                             )
                     except Exception as e:
-                        import pdb
+                        # import pdb
 
-                        pdb.set_trace()
+                        # pdb.set_trace()
```

Comment:
I think something is missing in this exception handler

Reply:
I'll leave the commented code in as a debugging convenience, but  I'll add a log as well so that things don't just get swallowed.